ui/app/components/policy-editor.hbs

```diff
@@ -34,7 +33,7 @@
           mode="ruby"
           content=@policy.rules
           onUpdate=this.updatePolicyRules
-          autofocus=(not @policy.isNew)
+          autofocus=false
```

Comment:
We decided to stop auto-focusing Edit Policy/Token/Role pages after internal feedback, as being automatically inside a textbox prevents using keyboard navigation to go back, etc.